test/calculus_tests.jl

```diff
@@ -676,10 +690,12 @@ function runtests()
                 nelement_local = nelement
 				nrank_per_block = 0 # dummy value
 				irank = 0 # dummy value
-				comm = MPI.COMM_NULL # dummy value 
+				comm = MPI.COMM_NULL # dummy value
+                element_spacing_option = "uniform"
```

Comment:
This test should work with a sqrt grid (the Chebyshev derivative tests just above should work too, but finding the expected errors is a faff, and those would probably change enough with the change of grid to make adding it there annoying), so remove this and add a
```julia
, element_spacing_option ∈ ("uniform", "sqrt")
```
to the test loop above.